crates/bevy_gilrs/src/lib.rs

```diff
@@ -3,21 +3,26 @@ mod gilrs_system;
 
 use bevy_app::prelude::*;
 use bevy_ecs::prelude::*;
-use gilrs_system::{gilrs_startup_system, gilrs_update_system};
+use gilrs::GilrsBuilder;
+use gilrs_system::{gilrs_event_startup_system, girls_event_system};
 
 #[derive(Default)]
 pub struct GilrsPlugin;
 
 impl Plugin for GilrsPlugin {
     fn build(&self, app: &mut AppBuilder) {
-        match gilrs::Gilrs::new() {
+        match GilrsBuilder::new()
+            .with_default_filters(false)
+            .set_update_state(false)
+            .build()
+        {
             Ok(gilrs) => {
                 app.add_thread_local_resource(gilrs)
-                    .add_startup_system(gilrs_startup_system.thread_local_system())
-                    .add_system_to_stage(
-                        stage::EVENT_UPDATE,
-                        gilrs_update_system.thread_local_system(),
-                    );
+                    .add_startup_system_to_stage(
```

Comment:
nit: this is equivalent to `add_startup_system(...)`, which is the approach we use everywhere else for "adding to the default stage".

Reply:
Done